base/worker_thread.hpp

```diff
@@ -33,11 +38,51 @@ class WorkerThread : public TaskLoop
   bool Push(Task && t) override;
   bool Push(Task const & t) override;
 
+  bool PushDelayed(Duration const & delay, Task && t);
```

Comment:
I think we need to document that it is not specified in which order the delayed tasks with coinciding "when" parameters will be executed.

Reply:
Also, this method needs to be documented anyway.

Reply:
Done